README.md

```diff
@@ -85,6 +85,11 @@ func (s *BasicStrategy) OnTick() error {
 	s.Exchange.OpenLong(symbol, OrderTypeLimit, 5000, 10)
 	s.Exchange.CloseLong(symbol, OrderTypeLimit, 6000, 10)
 
+	//Trailing Stop Market Order - sell stop loss order for long position
+	callbackRate := 5.0 // from 0.1% until 5% allowed
+	s.Exchange.PlaceOrder(symbol, Sell, OrderTypeTrailingStopMarket, 0.0, 10,
+		OrderCallbackRateOption(callbackRate), OrderReduceOnlyOption(true))
```

Comment:
Also **OrderActivationPriceOption** with activation price pass needed. see https://binance-docs.github.io/apidocs/futures/en/#new-order-trade → activationPrice

Reply:
@ksidenko actually `activationPrice` is optional. Default is latest price: "Used with TRAILING_STOP_MARKET orders, default as the latest price(supporting different workingType)". I tested without it and ti works fine. 
But I'll include it here in the README.